airflow/providers/ssh/operators/ssh.py

```diff
@@ -156,6 +167,8 @@ def raise_for_status(self, exit_status: int, stderr: bytes, context=None) -> Non
         if context and self.do_xcom_push:
             ti = context.get("task_instance")
             ti.xcom_push(key="ssh_exit", value=exit_status)
+        if self.skip_on_exit_code is not None and exit_status in self.skip_on_exit_code:
```

Comment:
It will never be None:
```python
self.skip_on_exit_code = (
      skip_on_exit_code
      if isinstance(skip_on_exit_code, Container)
      else [skip_on_exit_code]
      if skip_on_exit_code
      else []
  )
```
```suggestion
        if exit_status in self.skip_on_exit_code:
```

Reply:
Fixed